goss-testing/scripts/postgres_replication_lag.sh

```diff
@@ -1,19 +1,56 @@
 #!/bin/bash
 
 print_results=0
-while getopts ph stack
+while getopts pm:a:w:h stack
 do
     case "${stack}" in
           p) print_results=1;;
-          h) echo "usage: postgres_replication_lag.sh           # Only print 'PASS' upon success"
+          m) POSTGRES_MAX_LAG=$OPTARG;;
+          a) POSTGRES_MAX_ATTEMPTS=$OPTARG;;
+          w) POSTGRES_WAIT_SECONDS_BETWEEN_ATTEMPTS=$OPTARG;;
+          h) echo "usage: postgres_replication_lag.sh           # Print 'OK' and 'PASS' upon success"
              echo "       postgres_replication_lag.sh -p        # Print all results and errors if found. Use for manual check."
+             echo "       postgres_replication_lag.sh -p -m <max_allowed_lag> -a <number_attempts> -w <wait_seconds_between_attempts>"
+             echo "default parameters: -m 0  -a 10  -w 10"
              exit 3;;
-         \?) echo "usage: postgres_replication_lag.sh           # Only print 'PASS' upon success"
+         \?) echo "usage: postgres_replication_lag.sh           # Print 'OK' and 'PASS' upon success"
              echo "       postgres_replication_lag.sh -p        # Print all results and errors if found. Use for manual check."
+             echo "       postgres_replication_lag.sh -p -m <max_allowed_lag> -a <number_attempts> -w <wait_seconds_between_attempts>"
+             echo "default parameters: -m 0  -a 10  -w 10"
              exit 3;;
```

Comment:
Was this supposed to be changed to exit 4 since the FAIL case now uses exit 3?

Reply:
changed to exit 4